realsense2_description/urdf/_d405.urdf.xacro

```diff
@@ -0,0 +1,165 @@
+     publishing TF values with actual calibrated camera extrinsic values -->
+    <xacro:property name="d405_cam_depth_to_infra1_offset" value="0.0"/>
+    <xacro:property name="d405_cam_depth_to_infra2_offset" value="-0.018"/>
+    <xacro:property name="d405_cam_depth_to_color_offset" value="0.0"/>
+
+    <!-- The following values model the aluminum peripherial case for the
+  	d405 camera, with the camera joint represented by the actual
+  	peripherial camera tripod mount -->
+    <xacro:property name="d405_cam_width" value="0.042"/>
+    <xacro:property name="d405_cam_height" value="0.042"/>
+    <xacro:property name="d405_cam_depth" value="0.023"/>
+    <xacro:property name="d405_cam_mount_from_center_offset" value="0.01465"/>
+    <!-- glass cover is 0.1 mm inwards from front aluminium plate -->
+    <xacro:property name="d405_glass_to_front" value="0.1e-3"/>
+    <!-- see datasheet Revision 017, Table. 4-16 page 97 -->
+    <xacro:property name="d405_zero_depth_to_glass" value="3.7e-3"/>
+    <!-- convenience precomputation to avoid clutter-->
+    <xacro:property name="d405_mesh_x_offset" value="${d405_cam_mount_from_center_offset-d405_glass_to_front-d405_zero_depth_to_glass}"/>
+
+    <!-- The following offset is relative to the physical d405 camera peripherial
+  	camera tripod mount -->
+    <xacro:property name="d405_cam_depth_px" value="${d405_cam_mount_from_center_offset}"/>
+    <xacro:property name="d405_cam_depth_py" value="0.009"/>
+    <xacro:property name="d405_cam_depth_pz" value="${d405_cam_height/2}"/>
+
+    <!-- camera body, with origin at bottom screw mount -->
+    <joint name="${name}_joint" type="fixed">
+      <xacro:insert_block name="origin" />
+      <parent link="${parent}"/>
+      <child link="${name}_bottom_screw_frame" />
+    </joint>
+    <link name="${name}_bottom_screw_frame"/>
+
+    <joint name="${name}_link_joint" type="fixed">
+      <origin xyz="${d405_mesh_x_offset} ${d405_cam_depth_py} ${d405_cam_depth_pz}" rpy="0 0 0"/>
+      <parent link="${name}_bottom_screw_frame"/>
+      <child link="${name}_link" />
+    </joint>
+
+    <link name="${name}_link">
+      <visual>
+        <!-- the mesh origin is at front plate in between the two infrared camera axes -->
+        <origin xyz="${d405_zero_depth_to_glass + d405_glass_to_front} ${-d405_cam_depth_py} 0" rpy="${M_PI/2} 0 ${M_PI/2}"/>
+        <geometry>
+	        <mesh filename="file://$(find realsense2_description)/meshes/d405.stl" scale="0.001 0.001 0.001" />
+        </geometry>
+        <material name="aluminum"/>
+      </visual>
+      <collision>
+        <origin xyz="${d405_zero_depth_to_glass-d405_cam_depth/2} ${-d405_cam_depth_py} 0" rpy="0 0 0"/>
+        <geometry>
+          <box size="${d405_cam_depth} ${d405_cam_width} ${d405_cam_height}"/>
+        </geometry>
+      </collision>
+      <inertial>
+        <!-- The following are not reliable values, and should not be used for modeling -->
```

Comment:
The <inertial> section hasn't been updated. It has garbage values. 
In fact, all the previous URDF's of D415, D455, D435 have same garbage values.